src/PrestaShopBundle/Bridge/AdminController/ControllerConfiguration.php

```diff
@@ -223,21 +220,41 @@ class ControllerConfiguration
      * @param Employee $user the current user
      */
     public function __construct(
-        Employee $user
+        Employee $user,
+        int $tabId,
+        string $objectModelClassName,
+        string $legacyControllerName,
+        string $tableName,
+        string $templateFolder
     ) {
         $this->user = $user;
+        $this->tabId = $tabId;
+        $this->objectModelClassName = $objectModelClassName;
+        $this->legacyControllerName = $legacyControllerName;
+        $this->tableName = $tableName;
+        $this->templateFolder = $templateFolder;
     }
 
     /**
      * Adds toolbar action to the page
      *
-     * @param HeaderToolbarAction $action
+     * @param string $label
+     * @param array<string, mixed> $config
```

Comment:
Use [array shapes](https://phpstan.org/writing-php-code/phpdoc-types#array-shapes) when the structure is known.
```suggestion
     * @param array{href?: string, desc?: string, icon?: string, class?: string} $config
```